plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/index.tsx

```diff
@@ -195,7 +206,7 @@ const DatasetCard: React.FC<Props> = ({
                   </LeftMain>
                   <ArrowIcon icon="arrowDown" size={16} expanded={expanded} />
                 </HeaderContents>
-                {inEditor && expanded && (
+                {expanded && (
```

Comment:
The tabs should only be shown inEditor. Why did you remove the inEditor check here and with the add field and save buttons?

Reply:
This was made for developement purposes.
Done!